src/unittest/python/plugins/python/flake8_plugin_tests.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+from pybuilder.core import Project
+from mock import Mock
+from pybuilder.plugins.python.flake8_plugin import initialize_flake8_plugin
+
+
+class FlakePluginInitializationTests(TestCase):
+
+    def setUp(self):
+        self.project = Project("basedir")
+
+    def test_should_set_dependency(self):
+        mock_project = Mock(Project)
+        initialize_flake8_plugin(mock_project)
+        mock_project.build_depends_on.assert_called_with('flake8')
+
+    def test_should_leave_user_specified_properties_when_initializing_plugin(self):
+
+        expected_properties = {
+            "flake8_break_build": True,
+            "flake8_max_line_length": 80,
+            "flake8_exclude_patterns": ".svn",
```

Comment:
> Actually this might be confusing because the exclude pattern is not about file/folder names but about flake errors (W123 and so on)

sure?

http://pybuilder.github.io/documentation/plugins.html#Flake8plugin

> flake8_exclude_patterns 	string 	None, but flake8 comes with the default:
.svn,CVS,.bzr,.hg,.git,__pycache__ 	Comma separated list of file patterns to exclude
Example: ".git,__pycache__"